Marlin/src/inc/SanityCheck.h

```diff
@@ -2570,6 +2570,12 @@ static_assert(   _ARR_TEST(3,0) && _ARR_TEST(3,1) && _ARR_TEST(3,2)
   #error "BACKUP_POWER_SUPPLY requires a POWER_LOSS_PIN."
 #endif
 
+#if ENABLED(POWER_LOSS_ZHOME)
+  #if ENABLED(Z_SAFE_HOMING)
```

Comment:
for users who have a ZMAX endstop in addition to the ZMIN endstop it should be made an option to use that switch, but as I can see there is no way to ZHOME on opposite direction, when the switch is declared on the other side, so this could require some work. Also, that addition, could make it work on printers with Z_SAFE_HOMING